fe/fe-core/src/main/java/com/starrocks/analysis/LambdaFunction.java

```diff
@@ -0,0 +1,63 @@
+package com.starrocks.analysis;
```

Comment:
new Expression place in com.starrocks.sql.ast

Reply:
ok, done.